src/sinks/datadog/events/config.rs

```diff
@@ -105,11 +92,9 @@ impl DatadogEventsConfig {
 impl SinkConfig for DatadogEventsConfig {
     async fn build(&self, cx: SinkContext) -> crate::Result<(VectorSink, Healthcheck)> {
         let client = self.build_client(cx.proxy())?;
-        let healthcheck = self.dd_common.build_healthcheck(
-            client.clone(),
-            self.default_api_key.clone().into(),
-            self.region.as_ref(),
-        )?;
+        let healthcheck = self
+            .dd_common
+            .build_healthcheck(client.clone(), self.region.as_ref())?;
```

Comment:
Would it make sense to take this a step further and move `region` into `dd_common` as well?

Reply:
Ah, I had evaluated that on the initial PR for the shared struct, but the `region` is not part of the `datadog_traces` config. (region is considered deprecated and replaced by `site`)